api/api-doc/storage_service.json

```diff
@@ -1213,6 +1213,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"mixed_shard_optimization",
```

Comment:
I don't think we can call this an optimization. It is a workaround really. It throttles down repair so we don't hit the trashing.

Reply:
I do not mind the name too much. I changed it from reduce_reader_optimization to mixed_shard_optimization last minute before I sent this PR.

Reply:
It's aligned with 'small_table_optimization' naming convention.

Reply:
It should be triggered automatically, we can't rely on the user to know.

Reply:
Manager should know.

Reply:
It is not turned on for all mixed shard setup by default because the trade-off using one shard slows down the individual repair.